createlinks

```diff
@@ -48,10 +48,12 @@ event_templates("nethserver-nextcloud-save", qw(
   /etc/httpd/conf.d/default-virtualhost.inc
   /etc/httpd/conf.d/nextcloud.conf
   /etc/httpd/conf.d/zz_nextcloud.conf
+  /etc/opt/rh/rh-php72/php-fpm.d/000-nextcloud.conf
 ));
 
 event_services("nethserver-nextcloud-save", qw(
   httpd restart
+  rh-php72-php-fpm reload
```

Comment:
I didn't check, but is the reload enough for fpm to create the unix socket?
Usually to change the listen socket/port configuration, the daemons must be fully restarted.

Reply:
it will be created with nethserver-nextcloud-update, you commented on nethserver-nextcloud-save

Reply:
Ouch! My bad :(